pom.xml

```diff
@@ -344,6 +344,11 @@
 		<profile>
 			<id>spring</id>
 			<repositories>
+				<repository>
```

Comment:
Would you have to run this with `-P spring` spring profile enabled?

Reply:
This is already done when run with the maven wrapper: https://github.com/GoogleCloudPlatform/spring-cloud-gcp/blob/master/.mvn/maven.config